scripts/helper_functions.sh

```diff
@@ -2,6 +2,11 @@
 
 # Common functions definitions
 
+function get_php_version {
+    local PHPVER=`/usr/bin/php -r "echo PHP_VERSION;" | /usr/bin/cut -c 1,2,3`
+    echo ${PHPVER}
+}
+
 function install_php_sql_driver 
 {
```

Comment:
OK, good that this syntax error is fixed literally in a few minutes before I posted the comment...